forte/data/selector.py

```diff
@@ -85,14 +110,33 @@ def select(self, m_pack: MultiPack) -> Iterator[DataPack]:
                 f"Pack name {self.select_name}" f" not in the MultiPack"
             )
 
+    @classmethod
+    def default_configs(cls):
+        config = super().default_configs()
```

Comment:
I think `Configurable` class allows you to directly `return {"selector_name":None}`. No need to merge from `super().default_configs()` here since this step will be taken care of by `make_configs()`.

Reply:
Got itl